web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
+  schema "stripe_account" do
+    field :business_name, :string
+    field :business_url, :string
+    field :charges_enabled, :boolean
+    field :country, :string
+    field :default_currency, :string
+    field :details_submitted, :boolean
+    field :display_name, :string
+    field :email, :string
+    field :managed, :boolean
+    field :support_email, :string
+    field :support_phone, :string
+    field :support_url, :string
+    field :transfers_enabled, :boolean
+
+    belongs_to :organization, CodeCorps.Organization
+
+    timestamps()
+  end
+
+  @doc """
```

Comment:
Is this comment generated when creating this file? Imho, I don't believe `changeset` is in need of documentation and this could be removed.